tests/openvino/test_modeling.py

```diff
@@ -2079,6 +2186,19 @@ def get_preprocessors(self, model_arch):
             preprocessors = {"processor": processor, "tokenizer": None}
         return preprocessors
 
+    @parameterized.expand(SUPPORTED_ARCHITECTURES)
+    def test_model_can_be_loaded_after_saving(self, model_arch):
```

Comment:
What caused the need for this check. Can it be the case for Diffusion models as well?

Reply:
previously we had typo in model name for saving via save_pretrained is why I added this test. I suppose we have the same tests for diffusers (at least recently I added similar test for checking having safety_checker after loading)